src/plugins/shared/constants.ts

```diff
@@ -0,0 +1 @@
+export const GITHUBCOPILOT_FOR_AZURE_VSCODE_ID = "ms-azuretools.vscode-azure-github-copilot"
```

Comment:
💡 self note, this is very wider scope, I am removing this to the scope of use.

Reply:
This was moved